src/models/ItemFilterData.ts

```diff
@@ -21,6 +21,16 @@ export class ItemFilterData {
     return this.options
   }
 
+  get formattedFilterData(): FilterCheckboxGroup {
+    return {
+      id: this.field,
+      name: capitalize(this.field),
+      items: this.displayOptions().map((option) => {
```

Comment:
Maybe you can update `displayOptions` to return objects in this `{id, name}` format. The method  only used here and in one other place.

Reply:
Good call! Done